ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/backwardsync/ForwardSyncPhase.java

```diff
@@ -212,23 +217,15 @@ protected Void saveBlocks(final List<Block> blocks) {
         "Saved blocks {}->{}",
         () -> blocks.get(0).getHeader().getNumber(),
         () -> blocks.get(blocks.size() - 1).getHeader().getNumber());
+    batchSize = BackwardSyncContext.BATCH_SIZE;
     return null;
   }
 
   @VisibleForTesting
   protected CompletableFuture<Void> possiblyMoreForwardSteps(final BlockHeader firstNotSynced) {
     CompletableFuture<Void> completableFuture = CompletableFuture.completedFuture(null);
     if (firstNotSynced == null) {
-      final List<Block> successors = backwardChain.getSuccessors();
-      LOG.info(
-          "Forward Sync Phase is finished. Importing {} block(s) provided by consensus layer...",
-          successors.size());
-      successors.forEach(
-          block -> {
-            if (!context.getProtocolContext().getBlockchain().contains(block.getHash())) {
-              saveBlock(block);
-            }
-          });
+      LOG.info("Forward Sync Phase is finished. ");
```

Comment:
can these two LOG.info lines be combined?

Reply:
I removed it all together